scripts/orbit-versioner/orbitVersioner.ts

```diff
@@ -221,6 +225,40 @@ function _canBeUpgradedToTargetVersion(
       // cannot upgrade erc20 orbit chains from v1 to v3 right now due to a storage diff
       supportedSourceVersionsPerContract.Bridge = ['v2.0.0', 'v2.1.0', 'v2.1.2']
```

Comment:
Should we add 2.1.3 as a supported path to v3?

That should be availble in non fee token cases as well right?